src/Data/Erased.agda

```diff
@@ -0,0 +1,43 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Wrapper for the erased modality
+-- This allows us to store erased proofs in a record an use projections
+-- to manipulate them without having to turn on the unsafe option
+-- --irrelevant-projections.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Erased where
```

Comment:
Not sure about this, but I'm wondering if this might sit better under `Relation.Unary.Erased` rather than `Data.Erased`?

Reply:
It's not saying anything about a value of type `A`, it is wrapping one.
So I'd say it's definitely on the `Data` side of things.